CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+## General Workflow
+
+1. Fork the repo
```

Comment:
I think it might be easier for others if we just have one repo. Most, if not all of the contributors will be in our org anyway. We can just protect the master branch if needed.